torchrl/envs/common.py

```diff
@@ -220,6 +220,7 @@ def __init__(
     @classmethod
     def __new__(cls, *args, **kwargs):
         cls._inplace_update = True
+        cls.batch_locked = True
```

Comment:
This should not be modified, so let's turn in into a property and raise an error when someone wants to set this attribute